packages/effects-core/src/asset-manager.ts

```
@@ -150,14 +152,15 @@ export class AssetManager implements Disposable {
         ) {
           const { images: rawImages } = rawJSON.jsonScene;
           const images = scene.images;
+          const newImages: spec.ImageSource[] = [];
 
           for (let i = 0; i < rawImages.length; i++) {
             // 仅重新加载数据模板对应的图片
             if (images[i] instanceof HTMLCanvasElement) {
-              images[i] = rawImages[i];
+              newImages[i] = rawImages[i];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)